src/data-types/bigint.ts

```diff
@@ -30,7 +30,9 @@ const BigInt: DataType = {
     if (parameter.value == null) {
       return;
     }
-
+    // const buffer = Buffer.alloc(8);
+    // buffer.writeBigInt64LE(parameter.value);
+    // buffer.
```

Comment:
Forget to remove this comment, will remove it. Is this what we want for cleaning up the writing functions from writable buffer and use the native buffer writing functions instead?